### PR TITLE
Re-translate the writing style guide for zh-CN (Part 2) 

### PR DESCRIPTION
**Pull request by awxiaoxian2020**
continue #6238 .

Todo:

- [x] convert to md
- [x] add translations

### CODE REVIEW

**Comment by awxiaoxian2020** [comment]
@yin1999 Could you please help me to handle the unsupported `dl` elements?
``` markdown
 # Report from 2022/6/26 16:54:17
## All unhandled elements
- dl (12)
## Details per Document
### [/zh-CN/docs/MDN/Guidelines/Writing_style_guide](https://developer.mozilla.org/zh-CN/docs/MDN/Guidelines/Writing_style_guide)
### Missing conversion rules
- dl (326:1)
- dl (376:1)
- dl (389:1)
- dl (446:1)
- dl (457:1)
- dl (470:1)
- dl (483:1)
- dl (496:1)
- dl (511:1)
- dl (528:1)
- dl (543:1)
- dl (560:1)
```

**Comment by yin1999** [comment]
fixed, please using command `git rebase` to squash those two commits

**Comment by yin1999** [comment]
LGTM

**Comment by awxiaoxian2020** [comment]
> LGTM. Is it ready for merging?

I will add more translations when I'm available.

**Comment by yin1999** [comment]
> I will add more translations when I'm available.

Ok, please do this in the last commit

File: files/zh-cn/mdn/guidelines/writing_style_guide/index.md [hunk 1]
```diff
@@ -0,0 +1,468 @@
...
+  /* something */
+}
+
+var toolkitProfileService = Components.classes["@mozilla.org/toolkit/profile-service;1"]
+                           .createInstance(Components.interfaces.nsIToolkitProfileService);
+```
+
+#### 内联代码格式
+
+对于函数名、变量名、方法名等，应使用“Inline Code”按钮（编辑器中显示为“<>”的按钮）将其设置为内联代码格式（内联代码使用的是 {{HTMLElement("code")}} 元素）。比如：`frenchText()` 函数。
```
Comment (yin1999) [inline files/zh-cn/mdn/guidelines/writing_style_guide/index.md:199]: 请更新

**Review by yin1999 — CHANGES_REQUESTED** [review]

**Review by yin1999 — APPROVED** [review]
Thanks a lot

**Pull request merged**